ops/go.work

```diff
@@ -4,4 +4,3 @@ use (
 	.
 	../validation
 )
-
```

Comment:
What is the purpose of this go.work file? Is it necessary to keep it at this location instead of at the repo root?

Reply:
It's needed to allow local development of the validation package at the same time as the ops scripts. Otherwise we'd have to tag the validation package each time it changes.